README.md

```diff
@@ -0,0 +1,43 @@
+This file describes the work done and steps to perform test on the xacro package.
+
+"https://github.com/vandanamandlik/xacro/tree/ros2"
```

Comment:
I agree with @clalancette's comment above regarding refactoring the README.md.
In any case, we shouldn't be pointing to a fork of this repo.